src/utils/manager.py

```diff
@@ -59,18 +53,41 @@ class MountInfo:
     passno: str
 
 
-def _translate(url: str) -> Tuple[str, str]:
+def _translate(url: str) -> Optional[Tuple[str, Optional[int]]]:
     """Translate an NFS URL to a mount.nfs endpoint.
 
     Args:
         url: NFS uniform resource locator (URL)
 
     Returns:
-        Tuple[str, str]: `mount.nfs`-understandable endpoint and port number.
+        Optional[Tuple[str, Optional[int]]: `mount.nfs`-understandable endpoint and port number.
+        `None` if the url was invalid.
```

Comment:
[Nit]: Since we have the type annotation here for the return type, we don't need to include the return type here in docstring. As of recent, I usually don't include the return docstring since the type annotation indicates the possible branches that the API consumers needs to handle.

I'm fine with keeping this part of the docstring if you think it's necessary, but let's shorten it a bit.

```suggestion
        `mount.nfs`-understandable endpoint and port number. `None` if the NFS URL is invalid.
```

_I prefer to use present-tense in docstrings_. There are other docstrings that I want to revise here too, but that's beyond the scope of this PR and should be done later :smile: